### ISSUE TITLE
[주제 2] Invalid byte count

### ISSUE THREAD

**Issue by gyusang** [DatathonInfo/H.D.A.I.2021#31]
1. 아래 파일들에서 `LeadData`의 `LeadByteCountTotal`과 `WaveFormData`를 base64 decoding하여 얻은 byte 수가 일치하지 않습니다.
특히 byte 수가 홀수라서 16bit signed short로 decoding할 수 없습니다. 이런 데이터에 대한 해석은 어떻게 해야 할까요?

        electrocardiogram/data/validation/arrhythmia/8_2_007226_ecg.xml
        electrocardiogram/data/validation/arrhythmia/8_2_007281_ecg.xml
        electrocardiogram/data/train/arrhythmia/8_2_001879_ecg.xml
        electrocardiogram/data/train/arrhythmia/8_2_002164_ecg.xml
        electrocardiogram/data/train/arrhythmia/8_2_005580_ecg.xml

1. 1번처럼 Data 파일에 오류가 있는 경우 AUC 계산에서 이 데이터를 제외해야 할까요?

답변 기다리겠습니다.
감사합니다.

**Comment by DatathonInfo** [comment]
안녕하세요 참가자님,

해당 문의에 답변드립니다.

제공해 드린 원천 데이터는 인공지능 학습용 데이터 구축사업을 통해 제작된 데이터입니다.
본 대회의 목적은 구축된 원천 데이터를 통한 활용에 있어, 향후 개방될 원천 데이터를 그대로 제공해 드리고 있습니다.

제출 후 모델 평가가 진행되는 Test Set은 모두 정상적으로 lead가 포함되어 있는 데이터들로 구성되어 있습니다.
현재 문제가 확인된 파일;

Training Set;
![image](https://user-images.githubusercontent.com/92664643/144557734-e201e63b-7f0a-4d32-84c5-0ac138259271.png)

Validation Set;
![image](https://user-images.githubusercontent.com/92664643/144557685-ad8153bc-fcd3-4b29-a5cd-1e28a9a1d755.png)

별도로, 주관 측에 요청하여 문제되는 파일들을 확인중에 있습니다.

위에 제공된 문제 파일 외 문제가 되는 파일이 확인되시면 파일 제거 후 학습을 진행해 주시면 감사하겠습니다.

감사합니다.
H.D.A.I. 2021 운영팀 드림
